docs/_documentations/codewind-openapi-vscode.md

```diff
@@ -22,6 +22,21 @@ The Codewind OpenAPI Tools for VS Code provides commands that invoke the OpenAPI
 3. This extension pulls the [OpenAPI Generator CLI Docker Image](https://github.com/OpenAPITools/openapi-generator#16---docker) and runs the OpenAPI Generator in a Docker container. Install Docker if necessary.
 4. Install the Codewind OpenAPI Tools from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-openapi-tools) or by searching for `Codewind OpenAPI` in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
 
+## Generating and building code in an existing Java Maven project
+1. Right-click your Codewind project and launch the **Generate Server from OpenAPI Definition** wizard.
+2. Select your OpenAPI document `.json` file.
+3. Select the `jaxrs-spec` server generator type.
+4. When you're prompted to select the output folder, keep the default selection. The project root, which is the parent of the source folder, is selected by default.
```

Comment:
This is good.   Technically, you can generate into a child folder of a maven project, but the user will have to refactor the whole project to be multi-module.  This is beyond the scope of this enhancement.  We want to leave this option available in case they do want to do the codegen and refactor it themselves.